include/deal.II/base/parameter_handler.h

```diff
@@ -67,10 +67,19 @@ namespace Patterns
      */
     virtual bool match (const std::string &test_string) const = 0;
 
+    enum OutputStyle
+    {
+      // Machine interpretable format, default and used for create
+      Machine,
```

Comment:
As far as using upper case, I was matching the convention used in OutputStyle for ParameterHandler